src/Sweep.cpp

```diff
@@ -291,131 +291,246 @@ void InfectSweep(double t, int run) //added run number as argument in order to r
 	double fp; //// false positive
 	unsigned short int ts;
 
-	if (!P.DoSeasonality)	seasonality = 1.0;
-	else					seasonality = P.Seasonality[((int)t) % DAYS_PER_YEAR];
+	// if not doing seasonality
+	if (!P.DoSeasonality)
+	{	
+		// set seasonality to 1
+		seasonality = 1.0;
+	}
+	else{
+		// otherwise pick seasonality from P.Seasonality array using day number in year
+		seasonality = P.Seasonality[((int)t) % DAYS_PER_YEAR];
+	}
 
-	ts = (unsigned short int) (P.TimeStepsPerDay * t);
+	ts = (unsigned short int) (P.TimeStepsPerDay * t); // the timestep number of the start of the current day
+	// fp = false positive
```

Comment:
Another instance where I'd rather see the variable renamed as `false_positive` instead of a comment.